### ISSUE TITLE
The time cost for training this instructor?

### ISSUE THREAD

**Issue by afalf** [HKUNLP/instructor-embedding#14]
Hi, thanks for your great work. I do not see the time cost for training this model on the paper. Could you public the time cost for training the different size models(base, large, xl)?

**Comment by Harry-hash** [comment]
Hi, thanks a lot for your interest in INSTRUCTOR!

It takes around 6, 12 and 30 hours to train base, large and xl-sized models respectively. 

Feel free to leave any further question or comment here!

**Comment by afalf** [comment]
> Hi, thanks a lot for your interest in INSTRUCTOR!
> 
> It takes around 6, 12 and 30 hours to train base, large and xl-sized models respectively.
> 
> Feel free to leave any further question or comment here!

Thanks a lot for your reply. But I tried to run the train.py following README, but I find that it needs about 300 hours to train the gtr-base model under 4 A100 GPUs.... What should I do to check it?
<img width="474" alt="image" src="https://user-images.githubusercontent.com/101537385/219285768-61787601-1dae-4c01-9675-69757dd1c983.png">

**Comment by Harry-hash** [comment]
Hi, thanks a lot for your question!

We have prepared much more training data [here](https://drive.google.com/file/d/1_Ax8rTeeDKpNxj85LD2Yh7qsh3sYEiSH/view?usp=sharing). It may not be necessary to complete all the training steps. In our experiments, we stopped tuning INSTRUCTOR-base at around 40K steps.

Hope this helps! Feel free to leave any further question or comment here!

**Comment by afalf** [comment]
> Hi, thanks a lot for your question!
> 
> We have prepared much more training data [here](https://drive.google.com/file/d/1_Ax8rTeeDKpNxj85LD2Yh7qsh3sYEiSH/view?usp=sharing). It may not be necessary to complete all the training steps. In our experiments, we stopped tuning INSTRUCTOR-base at around 40K steps.
> 
> Hope this helps! Feel free to leave any further question or comment here!

Okay, thanks a lot!

**Comment by Harry-hash** [comment]
Feel free to reopen this issue and add any following comments!

**Comment by qmin2** [comment]
Was there a specific reason for ending the training at 40K steps? 
What criteria or basis(i.e. metric) did you use to determine the stopping point?

**Comment by yangjianxin1** [comment]
what is the batch size to train the instructor? Is 32 enough?
